packages/treasury/src/stablecoinMachine.js

```diff
@@ -97,24 +73,13 @@ export async function start(zcf, privateArgs) {
 
   assert(
     sameStructure(governedParams, harden(governedParameterLocal)),
-    X`Terms must match ${governedParameterLocal}`,
+    X`Terms ${governedParams} must match ${governedParameterLocal}`,
```

Comment:
Let me guess: you ran into this assertion yourself and discovered that it wasn't as nice as it could be :)